README.md

```diff
@@ -8,13 +8,18 @@
 [![GitHub Workflow Status for CI](https://img.shields.io/github/workflow/status/healthjoy/async-firebase/CI?label=CI&logo=github)](https://github.com/healthjoy/async-firebase/actions?query=workflow%3ACI)
 [![Codacy coverage](https://img.shields.io/codacy/coverage/b6a59cdf5ca64eab9104928d4f9bbb97?logo=codacy)](https://app.codacy.com/gh/healthjoy/async-firebase/dashboard)
 
-Async Firebase - is a lightweight asynchronous client to interact with Firebase.
 
   * Free software: MIT license
   * Requires: Python 3.6+
 
 ## Features
- TBD...
+
+  * Extremely lightweight and does not rely on ``firebase-admin`` which is hefty
+  * Send push notifications to Android and iOS devices
+  * Ability to set TTL (time to live) for notifications
+  * Ability to set priority for notifications
+  * Ability to set collapse-key for notifications
+  * Ability to test the request without actually delivering the message a.k.a dry-dun mode
```

Comment:
I meant we could get rid of `Ability to` in all bullet points so it would look like:
```
  * Extremely lightweight and does not rely on ``firebase-admin`` which is hefty
  * Send push notifications to Android and iOS devices
  * Set TTL (time to live) for notifications
  * Set priority for notifications
  * Set collapse-key for notifications
  * Dry-run mode for testing
```

Reply:
Well, do you think `Ability to` sounds oddly in this context? @aznbfg